Makefile

```diff
@@ -166,3 +166,6 @@ run-backend-celery:
 
 backend-command:
 	$(call backend_command,$(CMD))
+
+backend-manage-command:
```

Comment:
Probably we should remove this from the RP? I propose to only keep stuff that's relevant for the `install` issue